src/classes/NPSP_HouseholdAccount.cls

```diff
@@ -82,6 +82,10 @@ public inherited sharing class NPSP_HouseholdAccount {
                 null);
     }
 
+    public void isBillingAddressUndeliverable(Boolean isUndeliverable) {
```

Comment:
This method name is confusing to me, I expect "is..." methods to return a Boolean, not take one as a param.

Reply:
updated!